python/cudf/cudf/core/series.py

```diff
@@ -1417,7 +1417,7 @@ def as_mask(self):
         """
         return self._column.as_mask()
 
-    def astype(self, dtype, copy=False, errors="raise", **kwargs):
+    def astype(self, dtype, copy=False, errors="raise"):
```

Comment:
Apparently, the top level `astype` functions were never supposed to take `**kwargs` here. Although I don't see a reason it cant be retained in our internal column level helper functions. That said, it broke some of our tests, hence some of the `astype` test changes.